test/Tester/TestUtils.cs

```diff
@@ -114,20 +114,12 @@ public static class RequestContextTestUtils
     {
         public static void SetActivityId(Guid id)
         {
-#if NETSTANDARD
             RequestContext.ActivityId.Value = id;
```

Comment:
Since we are now officializing this approach (previously it was just a temp workaround for .NET Standard, so I didn't nit pick on it), it would be nice to not expose the AsyncLocal field, but instead have a wrapper property, so this would look like:
`RequestContext.ActivityId = id`

Same with the CallContextData field.